twitter_ads/audience.py

```diff
@@ -35,6 +37,44 @@ def create(klass, account, file_path, name, list_type):
             raise e
 
     @classmethod
+    def ta_file_upload(klass, account, file_path, name, list_type, expires_at, effective_at):
```

Comment:
We might as well include a sample file with dummy data if we are going to include a file based method - i.e. it's confusing whether or not the data in the file should be pre-hashed and if it's pre-hashed then maybe they need a utility to do that without mangling the other parts of the file (commas, etc)